### ISSUE TITLE
ImportError Zillow

### ISSUE THREAD

**Issue by agrushevsky** [ChrisMuir/Zillow#16]
import Zillow.zillow_functions as zl
ImportError: No module named 'Zillow'

**Comment by ChrisMuir** [comment]
Yeah the way the import step is written, the `zillow_runfile.py` is meant to be run from a working directory that contains the `Zillow` directory. If you set `Zillow` as your working directory, then in order to run it you just need to edit the import line to be like this:
```py
import zillow_functions as zl
```
